hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/verifier/BaseHandleContextVerifier.java

```diff
@@ -39,54 +42,76 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Helper class that contains all functionality for verifying signatures during handle.
+ * Base implementation of {@link HandleContextVerifier}
  */
-public class HandleContextVerifier {
```

Comment:
This change broke an existing integration test that isn't currently included in this PR:
```
hedera-node/hedera-app/src/itest/java/signatures/SignatureVerificationTest.java:95: error: HandleContextVerifier is abstract; cannot be instantiated
    final var handleContextVerifier = new HandleContextVerifier(hederaConfig, verificationResults);
```

Reply:
Thanks, fixed it.